src/Sarif/Visitors/GitHubIngestionVisitor.cs

```diff
@@ -12,47 +13,24 @@
         // Internal static rather than private const to allow a unit test with a practical limit.
         internal static int s_MaxResults = 5000;
 
+        private Run run;
+        private int ruleIndex = -1;
         private IList<Artifact> artifacts;
         private IList<ThreadFlowLocation> threadFlowLocations;
 
         public override Run VisitRun(Run node)
         {
+            this.run = node;
             this.artifacts = node.Artifacts;
             this.threadFlowLocations = node.ThreadFlowLocations;
 
             if (node.Results != null)
             {
-                int errorsCount = 0;
```

Comment:
>errorsCount [](http://example.com/codeflow?start=20&length=11)

I wouldn't delete this until you confirm that GH can accept warnings. Also, if this visitor was correct, the validator should also fire on any warnings in a log file (warning that they will be stripped).

Reply:
yes i think it supports warning, please see my other reply with screenshot.